### PR TITLE
TUT-2469 - fixing bug where version was wrong when trying to certify caliper events in MATHstream

### PR DESCRIPTION
**Pull request by pmcmahon-cl**

### CODE REVIEW

**Review by kschaefe — APPROVED** [review]

**Comment by kschaefe** [comment]
You'll need a version bump.  We'll likely want to change the package name as well.

**Comment by pmcmahon-cl** [comment]
@kschaefe  I've added another commit to change the name, version, and repository location in package.json. I don't have write access to this repo, so could you please merge the PR after you approve? Or grant me write access if possible?

**Review by kschaefe — APPROVED** [review]

**Pull request merged**
